mailpoet/lib/Entities/SubscriberEntity.php

```diff
@@ -447,6 +447,12 @@ public function getSegments() {
     });
   }
 
+  public function getSubscribedSegments() {
```

Comment:
> The name looks very similar to another getter getSubscriberSegments

Yeah, I thought about this but couldn't think of a better name and went it with. But I agree that it has the potential to induce mistakes.

> Maybe adding a parameter with status would be better because it allows better reusability.

What if we add a new optional `$status` parameter to `getSubscriberSegments()`? This way we don't even need to create `getSubscribedSegments()`?

Reply:
>What if we add a new optional $status parameter to getSubscriberSegments()? This way we don't even need to create getSubscribedSegments()?

This sounds great to me! 👍🏻

Reply:
@lysyjan, I implemented the approach that we discussed above in https://github.com/mailpoet/mailpoet/pull/4325/commits/422b422d451294e58814528cd6c2b0bf816dcd35. Could you please check this PR again?